src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -689,6 +689,7 @@ internal static double Round(double number, double digits, RoundType rt = RoundT
                 return number;
             }
 
+            // Dividing by m, since multiplication was introducing floating point error
```

Comment:
Maybe it's a github bug, but it looks like you removed the actual fix in the latest iteration?

Reply:
Yes, I noticed that. Fixing that!  Thanks!